.github/workflows/release.yml

```
@@ -341,21 +341,37 @@ jobs:
+
+          ## 📦 Artifacts
+          - Binary downloads include SHA-256 checksums
+          - SLSA Level 3 provenance attached
+          - SBOM files available in CycloneDX format
+
+          ## 📚 Documentation
+          - [Installation Guide](./install/README.md)
+          - [Security Verification](./docs/guides/security-verification.md)
+          - [SBOM Verification](./docs/guides/sbom-verification.md)
+          EOF
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)